kerlescan/view_helpers.py

```diff
@@ -46,6 +47,25 @@ def ensure_account_number(request, logger):
             )
 
 
+def ensure_has_role(
+    role, application, app_name, request, logger, request_metric, exception_metric
```

Comment:
Given the number of arguments that this method takes, there would be an advantage to using keyword parameters. That would make the readability of the caller easier.

Reply:
updated